### DISCUSSION TITLE
咱这个表格，没有 复选框功能吗？  想选中想要的多个不连续的 数据 咋弄？

### DISCUSSION THREAD

**Discussion by shifenhutu** [antvis/S2#2813] [General]
看官方文档， 原生似乎没有复选框的功能， 只能自己定制吗

**Comment by lijinke666** [comment]
没有.

按住 (Ctrl/Command) 可以对行列头进行多选 https://s2.antv.antgroup.com/examples/interaction/basic/#header-cell-click-selection

表格定位不同, 暂时没有复选框能力, 如有强诉求只能定制, 或 考虑其他库, 或 PR.
